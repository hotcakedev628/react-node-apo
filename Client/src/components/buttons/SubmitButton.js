import React from 'react'

export const SubmitButton =({label, disabled, classNames}) => (
    <button className={`btn btn-default ${classNames}`} disabled={disabled} type="submit"> {label || 'Submit'}</button>
)


export default SubmitButton