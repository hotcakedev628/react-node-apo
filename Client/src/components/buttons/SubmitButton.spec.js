import renderer from 'react-test-renderer';
import { SubmitButton } from './SubmitButton';
import React from 'react'


describe('SubmitButton', () => {
    it('should match snapshot', () => {
      const tree = renderer.create(<SubmitButton label="Submit" disabled={true} classNames="default"/>).toJSON();
      expect(tree).toMatchSnapshot();
    });
  });
  