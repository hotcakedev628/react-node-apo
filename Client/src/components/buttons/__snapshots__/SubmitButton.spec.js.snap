// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SubmitButton should match snapshot 1`] = `
<button
  className="btn btn-default default"
  disabled={true}
  type="submit"
>
   
  Submit
</button>
`;
